clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/TitusException.java

```diff
@@ -21,36 +21,16 @@
 public class TitusException extends IntegrationException {
   public TitusException(String message) {
     super(message);
-    setRetryable(true);
```

Comment:
Removed setting TitusException as retryable when only a message is provided - this makes sense to me given where this exception is thrown.